docs/coordinates/skycoord.rst

```diff
@@ -542,15 +542,14 @@ a spherical-type coordinate (and not, for example, a Cartesian coordinate).
 Furthermore, the frame's ``representation_component_names`` attribute defines
 the coordinate keyword arguments that |SkyCoord| will accept.
 
-Another important attribute is ``frame_attr_names``, which defines the
+Another important attribute is ``frame_attributes``, which defines the
 additional attributes that are required to fully define the frame::
 
   >>> sc_fk4 = SkyCoord(1, 2, frame='fk4', unit='deg')
-  >>> sc_fk4.get_frame_attr_names()
-  {'equinox': <Time object: scale='tt' format='byear_str' value=B1950.000>, 'obstime': None}
+  >>> sc_fk4.frame_attributes.keys()
+  dict_keys(['equinox', 'obstime'])
```

Comment:
Would it be better to demonstrate the full `dict` here not just the keys?

Reply:
Done